pyndl/count.py

```diff
@@ -75,7 +75,8 @@ def cues_outcomes(event_file_name,
     if verbose:
         print('\n...counting done.')
 
-    return n_events, cues, outcomes
+    cuesOutcomes = namedtuple('cuesOutcomes', 'n_events, cues, outcomes')
```

Comment:
Named tuples should be the same as a class / type definition and therefore capitalized:
```suggestion
    CuesOutcomes = namedtuple('CuesOutcomes', 'n_events, cues, outcomes')
```

Reply:
Do we want to have this as a definition that can be imported, i.e. on the base level?